arroyo-types/src/api.rs

```diff
@@ -0,0 +1,56 @@
+use serde::{Deserialize, Serialize};
```

Comment:
Can these be moved out of arroyo-types? Arroyo-types is the base crate for everything in our system, so only things that are needed by basically every part of the system (including UDFs compiled to WASM) should be here. It looks like this is just use by the api, so maybe can be moved there? 

We also try to keep dependencies minimal, so ideally we wouldn't add utoipa here.

Reply:
Yeah, I don't think we'll need them in the controller. The controller only interacts with the API via the database (i.e., the API writes stuff to the DB, the controller reads it and writes back some other stuff, which the API reads). And if we do, we can use a higher common dependency, like arroyo-rpc.